src/test/java/io/phasetwo/service/resource/OrganizationResourceTest.java

```diff
@@ -807,7 +790,7 @@ public void testAddGetDeleteRolesBulk() throws Exception {
           }
         };
     resp =
-        SimpleHttp.doPut(url, httpClient)
```

Comment:
I saw that the latest implementation: https://github.com/keycloak/keycloak/blob/ca0833b2e48ef3fac6c524bcb5b0680a73fdb6fa/server-spi-private/src/main/java/org/keycloak/broker/provider/util/SimpleHttp.java#L4 has a new implementation with signature doPut(String url, HttpClient client, long maxConsumedResponseSize)

Couldn't we use the default? 

    default long getMaxConsumedResponseSize() {
        return DEFAULT_MAX_CONSUMED_RESPONSE_SIZE;
    }


and not create a new class?     public static SimpleHttp doPut(String url, KeycloakSession session) {
        HttpClientProvider provider = session.getProvider(HttpClientProvider.class);
        return doPut(url, provider.getHttpClient(), provider.getMaxConsumedResponseSize());
    }

Reply:
That takes a `KeycloakSession` which we don't have. The one that takes an `HttpClient` is protected.

Reply:
Sorry . I didn't saw the access modifier.